bin/travis-ci/conf.vexpress_ca15_tc2_qemu

```diff
@@ -21,7 +21,7 @@
 console_impl=qemu
 qemu_machine="vexpress-a15"
 qemu_binary="qemu-system-arm"
-qemu_extra_args="-nographic -m 1G -tftp /tftpboot"
+qemu_extra_args="-nographic -m 1G -tftp ${UBOOT_TRAVIS_BUILD_DIR}"
```

Comment:
The x86 script creates a new e1000 network device (-device e1000) and user space network as backend for it. The arm ones already have a network device as part of their board definition and the -tftp option modifies properties of the user space netdev that is spawned by the board.